### ISSUE TITLE
print() in background fiber hogs fiber garbage collected memory pool

### ISSUE THREAD

**Issue by sir-Gollum** [tarantool/tarantool#72]
В expirationd попытался печатать удаляемые записи, в результате нашел такую проблему: чем больше print-ов, тем больше памяти занимает fiber. Тест ниже

init.lua:

function print10()
    box.fiber.detach()
    box.fiber.name('print10')
    a = 0
    while a < 10000 do
        a = a + 1
        print('this is a print10 test')  
    end  
    print('print10 done')
    while true do
        box.fiber.sleep(1)
    end
end

function print30()
    box.fiber.detach()
    box.fiber.name('print30')
    a = 0
    while a < 30000 do
        a = a + 1
        print('this is a print30 test')  
    end  
    print('print30 done')
    while true do
        box.fiber.sleep(1)
    end
end

print('running init')

f = box.fiber.create(print10)
box.fiber.resume(f)
f = box.fiber.create(print30)
box.fiber.resume(f)

print('init done')

Далее в консоли, сделав show palloc, видим:
    - name:  print30
      alloc: 3840128
      busy chunks:
        - { size: 32768, used: 118 }
    - name:  print10
      alloc: 1280128
